scripts/run_e2e_tests.py

```diff
@@ -411,8 +411,21 @@ def start_google_app_engine_server(dev_mode_setting):
     SUBPROCESSES.append(p)
 
 
+def get_chrome_driver_version():
+    output = os.popen('google-chrome --version').read()
+    chrome_version = ''.join(re.findall(r'([0-9]|\.)', output))
+    chrome_version = '.'.join(chrome_version.split('.')[:-1])
+    response = python_utils.url_open(
+        'https://chromedriver.storage.googleapis.com/LATEST_RELEASE_%s'
+        % chrome_version)
+    return response.read()
+
+
 def main(args=None):
     """Run the scripts to start end-to-end tests."""
+    global CHROME_DRIVER_VERSION
+
+    CHROME_DRIVER_VERSION = get_chrome_driver_version()
```

Comment:
Hm, don't you also need to change the `chromeDriver` property in protractor.conf.js to the corresponding version?

`chromeDriver: './selenium/chromedriver_xx.yy.zz',`

Reply:
Discussed. This is obsolete now.